packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -30,6 +63,81 @@ class SingleImageViewerContainer extends React.Component {
     })
   }
 
+  dragMove (event, difference) {
+    this.setState(prevState => {
+      const { viewBox } = Object.assign({}, prevState)
+      viewBox.x -= difference.x / 1.5
```

Comment:
Where's the `1.5` coming from, by the way? I noticed that the drag-move action doesn't move the image as much as the mouse action.

For example, if I start click-dragging at the right-edge of an image and drag all the way left (100% the length of the SVG width), the Subject image only moves left a distance of 50%  of the width of the SVG.

Reply:
Note: this was at natural zoom (Zoom In/Out wasn't clicked after Subject was loaded)

Reply:
Also, I'm not going to investigate touch events too much - there's still a bit of wonkiness with touch-drag causing page scroll.

Reply:
The `1.5` was an arbitrary smoothing factor so that we don't get jerky motion when dragging.

That's interesting about the page scroll. We've fixed that in PFE so it should not be re-introduced here.